modules/db_option_group/outputs.tf

```diff
@@ -0,0 +1,10 @@
+# DB option group
+output "this_db_option_group_id" {
+  description = "The db option group id"
+  value       = "${aws_db_option_group.this.id}"
```

Comment:
Use this instead: `value       = "${element(split(",", join(",", aws_db_option_group.this.*.id)), 0)}"`

Reply:
@antonbabenko  This PR looks useful. Were these changes to output the only thing holding the PR back?

Reply:
Yes, I think so but also this PR should be rebased to use the latest version of the module.